.github/workflows/kvrocks.yaml

```diff
@@ -438,11 +440,12 @@ jobs:
           wget https://github.com/Kitware/CMake/releases/download/v3.26.4/cmake-3.26.4-linux-x86_64.sh
           bash cmake-3.26.4-linux-x86_64.sh --skip-license --prefix=/usr
 
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3 #v4 use Node 20 and not working at CentOS 7
+      - uses: actions/setup-go@v4 #v5 use Node 20 too
```

Comment:
Let's create an issue for this.

Reply:
Of course, thanks!

Reply:
Let's talk - https://github.com/apache/kvrocks/issues/2101